tools/demos/trace-bind/demo.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Copyright 2016-2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+. $(dirname ${BASH_SOURCE})/../util.sh
+
+desc "Let's start a pod that's listening on TCP port 4242"
+run "kubectl run -n demo test-pod --image busybox -- sh -c 'while true ; do nc -l -p 4242 -w 1 ; done'"
+run "kubectl wait -n demo --for=condition=ready pod/test-pod"
+
+desc "Let's trace"
+run "kubectl gadget trace bind -n demo --timeout 3"
```

Comment:
Is it possible to add a sleep/delay at the end. I find that it terminates very quickly and one it not able to ready the result of the trace. Also having a comment indicating where the demo ends would be useful since it's played in a loop. (I suppose it applies for all gadgets too).